backend/chainlit/cli/__init__.py

```diff
@@ -128,19 +139,21 @@ async def start():
 )
 @click.option("--host", help="Specify a different host to run the server on")
 @click.option("--port", help="Specify a different port to run the server on")
-def chainlit_run(target, watch, headless, debug, ci, no_cache, host, port):
+def chainlit_run(target, watch, headless, debug, log_config, ci, no_cache, host, port):
     if host:
         os.environ["CHAINLIT_HOST"] = host
     if port:
         os.environ["CHAINLIT_PORT"] = port
+    if log_config:
```

Comment:
This makes not really sense here. Instead, the log_config should be added to class `RunSettings` and be set below:

```
config.run.debug = debug
config.run.log_config = log_config
```